db/default/users.rb

```diff
@@ -35,25 +37,30 @@ def prompt_for_admin_email
 
 def create_admin_user
   if ENV.fetch("AUTO_ACCEPT", true)
-    password = ENV.fetch("ADMIN_PASSWORD", "ofn123").dup
-    email = ENV.fetch("ADMIN_EMAIL", "ofn@example.com").dup
+    password = ENV.fetch("ADMIN_PASSWORD", "ofn123")
+    email = ENV.fetch("ADMIN_EMAIL", "ofn@example.com")
   else
-    puts 'Create the admin user (press enter for defaults).'
-    #name = prompt_for_admin_name unless name
+    Rails.logger.debug 'Create the admin user (press enter for defaults).'
```

Comment:
If user interaction is needed, I think `puts` is necessary here. (if you run it on a server environment then I think it will just hang waiting for input, without saying why..)

Reply:
We can actually use `say` here. Sorry, I missed that one.